docs/website/docs/dlt-ecosystem/verified-sources/rest_api.md

```diff
@@ -416,6 +416,7 @@ Available authentication types:
 | [BearTokenAuth](../../general-usage/http/rest-client.md#bearer-token-authentication) | `bearer` | Bearer token authentication. |
 | [HTTPBasicAuth](../../general-usage/http/rest-client.md#http-basic-authentication) | `http_basic` | Basic HTTP authentication. |
 | [APIKeyAuth](../../general-usage/http/rest-client.md#api-key-authentication) | `api_key` | API key authentication with key defined in the query parameters or in the headers. |
+| [OAuth2ClientCredentialsFlow](../../general-usage/http/rest-client.md#oauth20-authorization) | `oauth2` | OAuth 2.0 authorization with a temporary access token obtained from the authorization server. |
```

Comment:
```suggestion
| [OAuth2ClientCredentialsFlow](../../general-usage/http/rest-client.md#oauth20-authorization) | N/A | OAuth 2.0 authorization with a temporary access token obtained from the authorization server. |
```

We need to disable alias for know:
1. We could only make it available by updating rest_api source (https://github.com/dlt-hub/verified-sources/blob/master/sources/rest_api/config_setup.py#L75)
2. At the moment dictionary-based support is not available anyway since we require people to subclass the base OAuth class and adjust to their settings.

Reply:
Thank you!